sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/rel/BeamCalcRel.java

```diff
@@ -244,6 +244,7 @@ private RowCoder getErrorRowCoder(PCollection<Row> upstream, InputGetterImpl inp
     private final boolean verifyRowValues;
     private final List<String> jarPaths;
 
+    @SuppressWarnings("unused")
```

Comment:
This and the one in BeamZetaSqlCalcRel are both accessed by reflection. Would making them `public` fix this without the suppression?

Reply:
This looks like another place we should add TODO(BEAM-13271) (In the next errorprone releasethere will be a way to indicate that annotations like FieldAccess are used reflectively)

Reply:
I added a TODO(BEAM-13271) there